tiny_dnn/activations/softplus_layer.h

```diff
@@ -0,0 +1,49 @@
+/*
+    Copyright (c) 2017, Taiga Nomi
+    All rights reserved.
+
+    Use of this source code is governed by a BSD-style license that can be found
+    in the LICENSE file.
+*/
+#pragma once
+#include "tiny_dnn/activations/activation_layer.h"
+#include "tiny_dnn/layers/layer.h"
+
+namespace tiny_dnn {
+
+class softplus_layer : public activation_layer {
+ public:
+  using activation_layer::activation_layer;
+
+  std::string layer_type() const override { return "softplus-activation"; }
+
+  void forward_activation(const vec_t &x, vec_t &y) override {
+    for (serial_size_t j = 0; j < x.size(); j++) {
+      if (x[j] >= float_t(30)) {
```

Comment:
What is 30? Should it be a parameter similarly to LReLU? https://github.com/tiny-dnn/tiny-dnn/blob/master/tiny_dnn/activations/leaky_relu_layer.h#L89